src/docs/howto_contribute.md

```diff
@@ -115,13 +106,14 @@ You can always run the pre-commit checks manually via:
     :::bash
     $ ./build-support/bin/pre-commit.sh
 
-**Pro tip:** If you want your local master branch to be an exact copy of
+**Pro tip:** If you are certain that you have not accidentally committed anything to
```

Comment:
I might just remove the Pro tip entirely - seems out of scope for the Pants docs.